test/end-to-end/paths/intestacy/intestacy.en.test.js

```diff
@@ -80,7 +80,10 @@ Scenario(TestConfigurator.idamInUseText('Intestacy Child Journey - Digital iht,
     I.enterApplicantName('ApplicantFirstName', 'ApplicantLastName');
     I.enterApplicantPhone();
     I.enterAddressManually();
-    I.seeSummaryPage('*');
+    if (TestConfigurator.equalityAndDiversityEnabled()) {
+        I.exitEqualityAndDiversity();
+        I.completePCQ();
```

Comment:
Why do you need both of these steps?

Reply:
We need both methods as the first one performing back button navigation and completePCQ() method performing click action of "Save and continue" button.